src/backend.py

```diff
@@ -715,13 +715,13 @@ def load_multitrace(dirname, varnames=None, chains=None):
     if not istransd(varnames)[0]:
         logger.info('Loading multitrace from %s' % dirname)
         if chains is None:
-            files = glob(os.path.join(dirname, 'chain-*.csv'))
+            files = glob(os.path.join(dirname, 'chain*.csv'))
```

Comment:
whats the issue with the minus? this change like this will brake sth else, we need the minus

Reply:
else he tries to load in chain--* 
I am also confused where the addtional - comes from, so i just removed it here. Load in now works with this.